frontend/src/UserList.tsx

```diff
@@ -0,0 +1,133 @@
+import * as React from 'react';
+import {IPlayer} from './types';
+import styled from 'styled-components';
+import EmojiButton from './EmojiButton';
+import {WebsocketContext} from './WebsocketProvider';
+
+const Container = styled.div`
+  width: 80%;
```

Comment:
perhaps, 40% with min-width at 500px or so? On wide screens the table is really wide right now

Reply:
Updated in 554cb908cd44258732970504a2f6c557f3306593

On wide screen, looks like this:

<img width="1439" alt="Screen Shot 2020-04-19 at 3 01 04 AM" src="https://user-images.githubusercontent.com/935063/79681684-084fcf80-81ea-11ea-973a-4cb436048bb1.png">